collector/textfile.go

```diff
@@ -112,52 +230,20 @@ func (c *textFileCollector) parseTextFiles() []*dto.MetricFamily {
 				help := fmt.Sprintf("Metric read from %s", path)
 				mf.Help = &help
 			}
-			metricFamilies = append(metricFamilies, mf)
+			convertMetricFamily([]*dto.MetricFamily{mf}, ch)
```

Comment:
No need to pass in a slice of metric families now, just pass in `mf`. Then you can also remove the outer loop inside that function.